IDEAS/Fluid/Taps/TapProfile.mo

```diff
@@ -0,0 +1,84 @@
+within IDEAS.Fluid.Taps;
+model TapProfile
+  "Model for a DHW tap, reading the DHW demand profile via a CombiTimeTable."
+  extends IDEAS.Fluid.Taps.BaseClasses.PartialTap;
+
+  parameter Real uniCon=1/60
```

Comment:
I also prefer to let the users choose their preferred unit. Therefore, I used a conversion factor. Since DHW flow rates are typically specified in l/min, I prefer to keep the default units of the profiles in l/min and use a conversion factor of 1/60 to convert to kg/s.
Maybe @jelgerjansen can give his opinion?

Reply:
I would keep a conversion factor in there, keep its default value to be 1/60, and change the description of `con` (which now states that the conversion factor is used to change from l/min to kg/s).

Reply:
https://github.com/open-ideas/IDEAS/pull/1355/commits/c0d61bb4b1d3f00dc8ede996cff0c154c1efab33